Tasks/JenkinsDownloadArtifactsV1/AzureStorageArtifacts/blobservice.ts

```diff
@@ -16,27 +13,6 @@ export class BlobService {
         this._host = host;
     }
 
-    public async uploadBlobs(source: string, container: string, prefixFolderPath?: string, itemPattern?: string): Promise<string[]> {
```

Comment:
We moved the file but don't need this? Is it dead code?

Reply:
Yeah. There was actually a fair amount of dead code here, I didn't move several files